app/routes/index/chapter/welcome.js

```diff
@@ -1,4 +1,13 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  paginationNav: Ember.inject.service('pagination-nav'),
+  beforeModel() {
```

Comment:
This isn't related to the model, so it should be refactored into another part of the route lifecycle at a later time. Or, as mentioned in the commit comments, extracted into something that runs on all routes.